documentation/src/main/asciidoc/topics/data_container/proc_setting_cache_size.adoc

```diff
@@ -0,0 +1,23 @@
+[id='setting_cache_size_{context}']
+= Setting the Cache Size
```

Comment:
I am not sure if this needs to be a procedure at all. I can see this become a reference module with a table with ways of settings size:

* By count of items in the cache (`COUNT` is not very helpful, and I had to read the full sentence to grasp what it was referring to)
* By memory allocation (ditto)